tests/test_config_reader.py

```diff
@@ -293,6 +300,7 @@ def test_construct_stacks_constructs_stack(
             external_name=None,
             notifications=None,
             on_failure=None,
+            disable_rollback=False,
             stack_timeout=0,
             required_version=">1.0",
             template_bucket_name="stack_group_template_bucket_name",
```

Comment:
You should also add two tests that shows disable_rollback being set to True when it's present on the config and in the command params.

Reply:
yes sir.

Reply:
tests added